content/en/platform/corda/4.8/open-source/tools-index.md

```diff
@@ -24,6 +24,6 @@ wish to try the [Blob Inspector](blob-inspector.md).
 
 * [Corda Network Builder](network-builder.md)
 * [Network Bootstrapper](network-bootstrapper.md)
-* [DemoBench - **Warning** - DemoBench is deprecated in Corda 4.8](demobench.md)
+* [DemoBench - Warning: DemoBench is deprecated in Corda 4.8](demobench.md)
```

Comment:
```suggestion
```
I must have missed this one, as there was another ticket to remove DemoBench references in 4.7+

Reply:
You might get a merge conflict as I had already deleted this line...